ChangeLog

```diff
@@ -1,5 +1,10 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+mbed TLS 2.x.x branch released xxxx-xx-xx
+
+Feature
+    * Enhance support for signature-hash-algorithm extension.
```

Comment:
The changelog entry is also missing credit to the initial reporter (following the model of other changelog entries).